cat-boot-report-pdf/src/main/java/cc/catalysts/boot/report/pdf/elements/ReportTable.java

```diff
@@ -175,7 +179,7 @@ private float calculateVerticalAlignment(ReportElement[] line, int elementIndex,
         return yPos;
     }
 
-    private float printLine(PDDocument document, PDPageContentStream stream, int pageNumber, float startX, float y, float allowedWidth, ReportElement[] line) throws IOException {
+    private float printLine(PDDocument document, PDPageContentStream stream, int pageNumber, float startX, float y, float allowedWidth, ReportElement[] line, float previousLineHeight) throws IOException {
```

Comment:
You are introducing a new argument here, that has no other related changes, that's confusing.